### PR TITLE
Feature/prodes

### PR DESCRIPTION
**Pull request by GabrielMarianosf**
ADD filter prodes

### CODE REVIEW

**Comment by GabrielMarianosf** [comment]
[PRODES](https://trello.com/c/NPBKY5Aj/7-prodes)

File: components/support/SupportLayerFiltersProdes.vue [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+                        :required="true"
+                        outlined
+                        dense
+                    />
+                </v-col>
+                <v-col cols="6">
+                    <BaseDateField
+                        v-model="filters.end_date"
+                        :label="$t('end-date-label')"
+                        :required="true"
```
Comment (dagnaldo) [inline components/support/SupportLayerFiltersProdes.vue:18]: only `required` does not works?

Comment (dagnaldo) [inline components/support/SupportLayerFiltersProdes.vue:18]: only required does not works?

File: components/support/SupportLayerFiltersProdes.vue [hunk 2]
```diff
@@ -0,0 +1,221 @@
+<template>
+    <v-form v-if="layer.layer_filters.length" v-model="valid">
+        <v-row dense class="my-4">
+            <template v-if="hasDoubleDate">
+                <v-col cols="6">
+                    <BaseDateField
+                        v-model="filters.start_date"
+                        :label="$t('start-date-label')"
+                        :required="true"
```
Comment (dagnaldo) [inline components/support/SupportLayerFiltersProdes.vue:9]: only `required` does not works?

Comment (dagnaldo) [inline components/support/SupportLayerFiltersProdes.vue:9]: only `required` does not works?

File: components/support/SupportLayerFiltersProdes.vue [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+                    hasStartDate = true
+                } else if (layerFilter.filter_type === 'end_date') {
+                    hasEndDate = true
+                }
+            })
+
+            this.hasDoubleDate = hasStartDate && hasEndDate
+        }
+    },
+    mounted() {
```
Comment (dagnaldo) [inline components/support/SupportLayerFiltersProdes.vue:174]: Missing blank line.

Comment (dagnaldo) [inline components/support/SupportLayerFiltersProdes.vue:174]: Missing blank line.

File: components/support/SupportLayerFiltersProdes.vue [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+        this.getFilterOptions()
+    },
+    watch: {
```
Comment (dagnaldo) [inline components/support/SupportLayerFiltersProdes.vue:177]: Missing blank line.

Comment (dagnaldo) [inline components/support/SupportLayerFiltersProdes.vue:177]: Missing blank line.

File: components/support/SupportLayerFiltersProdes.vue [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+        'filters.co_cr'(value) {
+            this.populateTiOptions(value)
+        },
+    },
+    computed: {
```
Comment (dagnaldo) [inline components/support/SupportLayerFiltersProdes.vue:182]: Missing blank line.

Comment (dagnaldo) [inline components/support/SupportLayerFiltersProdes.vue:182]: Missing blank line.

File: components/support/SupportLayersGroupItemProdes.vue [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+            </v-list-item-avatar>
+            <v-list-item-content>
+                <v-list-item-title class="text-wrap">
+                    <span class="text-cursor">
+                        {{ layer.name }}
+                    </span>
+                </v-list-item-title>
+            </v-list-item-content>
+
+            <!-- <v-list-item-action v-if="disabledHeatmap">
```
Comment (dagnaldo) [inline components/support/SupportLayersGroupItemProdes.vue:34]: Remove unecessary comments.

Comment (dagnaldo) [inline components/support/SupportLayersGroupItemProdes.vue:34]: Remove unecessary comments.

File: store/supportLayers.js [hunk 7]
```diff
@@ -118,6 +126,34 @@ export const mutations = {
             Vue.set(state.supportCategoryGroupsFire, group.id, group)
         }
     },
+    setSupportCategoryGroupsProdes(state, categoryGroups) {
```
Comment (dagnaldo) [inline store/supportLayers.js:129]: Missing blank line between mutations.

Comment (dagnaldo) [inline store/supportLayers.js:129]: Missing blank line between mutations.

Comment (dagnaldo) [reply store/supportLayers.js:129]: Check all occurences.

Comment (dagnaldo) [reply store/supportLayers.js:129]: Check all occurences.

**Review by dagnaldo — CHANGES_REQUESTED** [review]

File: store/supportLayers.js (hunk 7 above)
Comment (GabrielMarianosf) [reply store/supportLayers.js:130]: Resolved.

Comment (GabrielMarianosf) [reply store/supportLayers.js:130]: Resolved.

File: components/support/SupportLayersGroupItemProdes.vue (hunk 6 above)
Comment (GabrielMarianosf) [reply components/support/SupportLayersGroupItemProdes.vue]: Resolved.

Comment (GabrielMarianosf) [reply components/support/SupportLayersGroupItemProdes.vue]: Resolved.

File: components/support/SupportLayerFiltersProdes.vue (hunk 2 above)
Comment (GabrielMarianosf) [reply components/support/SupportLayerFiltersProdes.vue]: Resolved.

Comment (GabrielMarianosf) [reply components/support/SupportLayerFiltersProdes.vue]: Resolved

File: components/support/SupportLayerFiltersProdes.vue (hunk 1 above)
Comment (GabrielMarianosf) [reply components/support/SupportLayerFiltersProdes.vue]: Resolved.

Comment (GabrielMarianosf) [reply components/support/SupportLayerFiltersProdes.vue]: Resolved.

File: components/support/SupportLayerFiltersProdes.vue (hunk 5 above)
Comment (GabrielMarianosf) [reply components/support/SupportLayerFiltersProdes.vue:184]: Resolved.

Comment (GabrielMarianosf) [reply components/support/SupportLayerFiltersProdes.vue:184]: Resolved.

File: components/support/SupportLayersGroupItemProdes.vue (hunk 6 above)
Comment (GabrielMarianosf) [reply components/support/SupportLayersGroupItemProdes.vue]: Resolved.

File: pages/index/support-prodes.vue [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+    },
+    "pt-br": {
+        "title": "Camadas"
+    }
+}
+</i18n>
+
+<script>
+import { mapState } from 'vuex'
+import _ from 'lodash'
```
[the remark this answers is not in the document]
Comment (GabrielMarianosf) [reply pages/index/support-prodes.vue:52]: Yes, in the ordersSupportLayersGroups method, lodash's sortBy function is used to sort groups and layers based on order property.

**Pull request merged**